test/e2e/seeder/ganache-seeder.js

```diff
@@ -1,15 +1,14 @@
 const { ethers } = require('ethers');
-const ganache = require('ganache');
-const { contractConfiguration } = require('./smart-contracts');
+const { SMART_CONTRACTS, contractConfiguration } = require('./smart-contracts');
 const GanacheContractAddressRegistry = require('./ganache-contract-address-registry');
 
 /*
  * Ganache seeder is used to seed initial smart contract or set initial blockchain state.
  */
 class GanacheSeeder {
-  constructor(debug = false) {
-    this.debug = debug;
```

Comment:
removed debug in the ganache seeder as we can simply add the property `quiet: false` to ganache object in our tests